src/platform/utilities/ui/asyncLoader.jsx

```diff
@@ -4,8 +4,7 @@ import LoadingIndicator from '@department-of-veterans-affairs/formation-react/Lo
 
 export default function asyncLoader(getComponent, message) {
   return class AsyncComponent extends React.Component {
-    static Component = null;
```

Comment:
Why were you removing the static Component member here? This changes how the component works. State is specific to an instance of a component, static members are shared across any use of this class.

Reply:
Because the `Component` seems to be declared twice. Here and in the state.

Reply:
It is. It's declared in state because that is the expected way to trigger a re-render when some data changes, to update the state. It's stored as a static member so that if the component returned from `asyncLoader` is instantiated multiple times, it doesn't try to re-fetch the component code.